.github/workflows/issues.yml

```diff
@@ -0,0 +1,16 @@
+name: Add issues to main project board
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@main
+        with:
+          project-url: https://github.com/orgs/phylum-dev/projects/24
+          github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
```

Comment:
Where does this secret come from? It does not appear in the [secrets settings for the repo](https://github.com/phylum-dev/phylum-ci/settings/secrets/actions) as an organization, environment, or repository secret.

Reply:
It's a repository secret. 
![image](https://user-images.githubusercontent.com/138794/172236575-717ceb7d-99ef-4d22-a68e-75fdd6935eb9.png)

Yeah if you can tackle adding the extra commit, that'd be great @maxrake!